tunedev/Banka
# #165559725 improve user interface

#### What does this PR do?
This pr improves the user interface by adding some changes

#### Description of Task to be completed?
> - Replace transaction full details page with a modal
> - Change typeface combination
> - Add page for the admin to create a new staff

#### How should this be manually tested?
> - Clone this repo
> - Checkout into this branch
> - Navigate into the ui folder
> - Open the index.html on google chrome(or any modern browser)
> - The pages should look better than it was

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165559725](https://www.pivotaltracker.com/story/show/165559725)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
